src/main/java/org/mockbukkit/mockbukkit/entity/LivingEntityMock.java

```diff
@@ -103,11 +107,20 @@ public abstract class LivingEntityMock extends EntityMock implements LivingEntit
 	private boolean collidable = true;
 	private boolean ai = true;
 	private boolean swimming;
+	/**
+	 * Set whether this entity is slumbering.
+	 */
+	@Getter @Setter
 	private boolean sleeping;
+	/**
+	 * Set whether this entity is climbing.
+	 */
+	@Getter @Setter
 	private boolean climbing;
 	private double absorptionAmount;
 	private int arrowCooldown;
 	private int arrowsInBody;
+	@Getter @Setter
 	private @Nullable Player killer;
 
 	private final Set<ActivePotionEffect> activeEffects = new HashSet<>();
```

Comment:
Should we replace this with a `Map<String, ActivePotionEffect>` to improve the code readability and performance?

Reply:
Thanks, I adjusted this now to use a map. I think it improves the performance, but code-readability-wise I feel the benefits are negligible.